tests/AtelierTomato.Markov.Core.Test/MarkovTests.cs

```diff
@@ -26,7 +26,7 @@ public async Task DatabaseFailure()
 		{
 			var options = Options.Create(new MarkovChainOptions { });
 			var sentenceAccess = Mock.Of<ISentenceAccess>();
-			var filter = new SentenceFilter(null, null);
+			var filter = new SentenceFilter([], []);

```

Comment:
why did you replace the null case? you could have kept the null case and added the empty case.